File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int,int>freq;
        for(int i=0;i<nums.size();i++)
        {
            freq[nums[i]]++;
        }
        vector<pair<int,int>>arr;
        for(auto n:freq)
        {
            arr.push_back({n.second,n.first});
        }
        sort(arr.rbegin(),arr.rend());
        
        vector<int>res;
        for(int i=0;i<k;i++)
        {
            res.push_back(arr[i].second);
        }
        return res;

    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        result=[0]*len(nums)
        for i in range(len(nums)):
            res=1
            for j in range(len(nums)):
                if i==j:
                    continue
                else:
                    res*=nums[j]
            result[i]=res
        return result